src/test/java/org/sagebionetworks/bridge/adherence/WeeklyAdherenceReportWorkerProcessorTest.java

```diff
@@ -305,6 +319,163 @@ public void limitsScanToProvidedAppsAndStudies() throws Exception {
         verify(mockWorkersApi, never()).getWeeklyAdherenceReportForWorker("app1", "study1b", "user1");
         verify(mockWorkersApi, never()).getWeeklyAdherenceReportForWorker("app2", "study2b", "user1");
     }
+    
+    @SuppressWarnings("unchecked")
+    @Test
+    public void onlyRunsOnTargetedHourOfDay() throws Exception {
```

Comment:
It looks like this test includes a lot of things that are already covered by the previous test. Can we simplify this test?

Reply:
I've tried to create tests that are just demonstrating the use of the time zones and reporting hours, and taken out code that verifies the request of apps and users. Also I've renamed those tests and added more explicit comments about what they're testing. Hopefully it's clearer.